index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link href="https://fonts.googleapis.com/css?family=Poppins:400,600" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <Header>
+        <div class="logo">My Logo</div>
+        <div class="menu"><img src="images/Union.png" alt="menu"></div>
```

Comment:
- [ ] The logo should be wrapped in `a` tag see  screenshot 👇
![a](https://user-images.githubusercontent.com/61562626/129186867-510f68d9-2f94-4c42-9c18-734cbbe9613d.jpeg)